plugins/grr/help.md

```diff
@@ -335,19 +335,23 @@ Example input:
 |results|string|False|Issues found with TruffleHog|
 
 ### Triggers
+  
+*This plugin does not contain any triggers.*
+### Tasks
+  
+*This plugin does not contain any tasks.*
 
-_This plugin does not contain any triggers._
-
-### Custom Output Types
-
-_This plugin does not contain any custom output types._
+### Custom Types
+  
+*This plugin does not contain any custom output types.*
 
 ## Troubleshooting
-
-_This plugin does not contain any troubleshooting information._
+  
+*There is no troubleshooting for this plugin.*
 
 # Version History
 
+* 3.0.1 - Updated plugin to set the status as obsolete
```

Comment:
(Suggestion) -> Updated plugin to set the status to obsolete. This plugin is no longer supported.

Reply:
Thanks 🙂  updated.